tests/README.md

```diff
@@ -0,0 +1,66 @@
+## Requirements
+- pytest
+
+## Run tests
+- normal (use GPU if it's available)
+    ```bash
+    # in root directory of this package
+    $ python -mpytest ./tests
+    ```
+
+- run tests on a machine without GPU
+    ```bash
+    # in root directory of this package
+    $ CPU_ONLY=1 python -mpytest ./tests
+    ```
```

Comment:
This is a good idea. A possible improvement would be to make it a pytest command-line argument instead of using environment variables.

Also, I think 'run tests on a machine without GPU' doesn't reflect the use-case. On a machine without GPU the normal execution will automatically pick the CPU. The use-case is more a  'force CPU-only execution'

Reply:
Ya, you're right. `CPU_ONLY` is more likely a term indicating 'force CPU-only execution'. I'll revise the description.

And it seems nice to replace it with a pytest command-line argument. But since it isn't used by fixtures directly, I'm planning to add a custom config object to store any further command line arguments, and bind it to `pytest` namespace dynamically. I found this approach works since [`pytest.config` global object is deprecated in pytest 5.0](https://docs.pytest.org/en/latest/deprecations.html#pytest-config-global).

```python
# In conftest.py

def pytest_addoption(parser):
    parser.addoption(
        '--cpu_only', action='store_true',
        help='Forcibly run all tests on CPU.'
    )

def pytest_configure(config):
    # bind a config object to `pytest`
    pytest.custom_cmdopt = CustomCommandLineOption()

    # add command line argument to that config object
    pytest.custom_cmdopt.add(
        'cpu_only', config.getoption('--cpu_only')
    )
```

```python
# In task.py
import pytest

def use_cuda():
    if pytest.custom_cmdopt.cpu_only:
        return False
    else:
        return torch.cuda.is_available()
```